### DISCUSSION TITLE
Dark background for launch screen when using dark themes

### DISCUSSION THREAD

**Discussion by euivanw** [Kong/insomnia#3721] [Ideas]
- Insomnia Version: 6.4.2
- Operating System: Windows 10

## Details

Hi Guys!

I'm using Insomnia with a dark theme, however, every time that I openned Insomnia, I get a white background color screen while Insomnia is loading the application. Is it possible to change that background color from white to a dark color, maibe related to the theme I choose?

![Opening](https://user-images.githubusercontent.com/371218/57074453-4642a380-6cba-11e9-9f78-d29fa1c35ceb.png)

**Comment by welcome** [comment]
👋 Thanks for opening your first issue! If you're reporting a 🐞 bug, please make sure
you include steps to reproduce it. If you're requesting a feature 🎁, please provide real
use cases that would benefit. 👪

To help make this a smooth process, please be sure you have first read the
[contributing guidelines](https://github.com/getinsomnia/insomnia/blob/master/CONTRIBUTING.md).

**Comment by gschier** [comment]
I took a brief look at how hard this would be to do and it's a bit trickier than I expected. Going to leave this one open for a later time.

**Comment by karthik-durai** [comment]
Can I take this?
